toolkit/tools/internal/logger/log.go

```diff
@@ -99,10 +101,16 @@ func InitBestEffort(path string, level string) {
 		level = defaultStderrLogLevel.String()
 	}
 
-	InitStderrLog()
+	_, callerFilePath, _, ok := runtime.Caller(parentCallerLevel)
```

Comment:
Should this code be moved into `initStderrLogInternal` to avoid duplicating it? (I think if you set `parentCallerLevel` to `2` this should work.)

Reply:
Ah, right, I can move the logic for the `toolName` variable.

I intentionally repeat the `runtime.Caller` code, because I want that call to be present only inside the public functions, to have a guarantee we're getting the file path of the actual use of the logger. Moving it deeper makes the code easier to break during refactoring or extension in the future.